src/processor/map/map_copy_from.cpp

```diff
@@ -164,9 +164,11 @@ std::unique_ptr<PhysicalOperator> PlanMapper::createCopyRel(
         offsetVectorIdx, std::move(columnTypes));
     auto printInfo =
         std::make_unique<RelBatchInsertPrintInfo>(copyFrom.getInfo()->tableEntry->getName());
+    auto progressSharedState = std::make_shared<RelBatchInsertProgressSharedState>();
```

Comment:
I don't think you need this if you move the logic inside `RelBatchInsert::initGlobalStateInternal`.

Reply:
hmm I thought you want to revert the change here? Don't you initialize the progressSharedState inside `   initGlobalStateInternal` already?

You can keep a private constructor for `RelBatchInsert` that's only for `clone`.

Reply:
There are no instance of progressSharedState to construct the RelBatchInsert class in this part of the code, so I think that I need to make a dummy copy here (From my understanding it's not getting used?) Would this pose a problem?